package.json

```diff
@@ -22,10 +22,15 @@
     "create-react-native-app": "^1.0.0",
     "expo": "^27.0.1",
     "firebase": "^5.4.2",
+    "native-base": "^2.8.0",
```

Comment:
It seems like you aren't using this module. Please remove if you are not using it. 🙏